### ISSUE TITLE
Picture Chooser returns wrong Image Size on iPhone

### ISSUE THREAD

**Issue by baumerik** [MvvmCross/MvvmCross-Plugins#29]
for the PictureChooser-Plugin you can set a maxDimension parameter.
On resizing the image on iOS the returned image does not get the specified size.

Research:
in the Resize method the image use the system defined scaling factor. iPhone 5 we get as Scalefactor 2.0 and so the image was not 1280x960 as expected. it returns 2560x1920.

**Comment by Cheesebaron** [comment]
I don't think this is a 'fix'. It is intentional to use the display scale to get the correct sized images depending on the screen dpi.

Maybe a better solution would be to give the option to the dev on what to do instead of hardcoding this value.

**Comment by baumerik** [comment]
But what is the main use case of this plugin?
Take a Picture an directly display it to the user?
In our use case is that we take the picture. Then maybe we will show it to the user but the main target is to send it to a Webserver. On all the other devices the correct image size is returned. 
So this a behavior on iOS that is a little bit different than the other platforms.

What is your preferred alternative way? Make this as configurable plugin for iOS? Extend the api for the plugin and so for all platforms? Or should we take the picture and resize it again if the size is not as expected?
